### DISCUSSION TITLE
Handling IF or Switch Statements

### DISCUSSION THREAD

**Discussion by Gaztab** [microsoft-search/pnp-modern-search#2341] [General]
Hi,
I am using PnP Search Results in Card view, I am trying use a Switch statement or IF statement within the 'Location' field. Can the results cards handle them?

I want to basically do something like
IF{{RefinableString00}} = "Current", <b style="color:green !important"</b>, <b style="color:red !important"</b>

Does this make sense?

Thanks

**Comment by wobba** [comment]
You need to look into the handlebar `if` statement (https://handlebarsjs.com/guide/builtin-helpers.html) or maybe the `eq` helper (https://github.com/helpers/handlebars-helpers#eq)

**Reply by Gaztab** [reply]
Thank you again, this pointed me in the right direction. I've since used it along with a few other guides to help put together a solution.
